server/market/orderrouter.go

```diff
@@ -593,6 +618,29 @@ func (r *OrderRouter) handleMarket(user account.AccountID, msg *msgjson.Message)
 	return nil
 }
 
+// Check the FundingCoin confirmations, and if zero, ensure the tx fee rate
+// is sufficient, > 90% of our last recorded estimate for the asset.
+func (r *OrderRouter) checkZeroConfs(dexCoin asset.FundingCoin, fundingAsset *asset.BackedAsset) *msgjson.Error {
+	// Verify that zero-conf coins are within 10% of the last known fee
+	// rate.
+	confs, err := coinConfirmations(dexCoin)
+	if err != nil {
+		log.Debugf("Confirmations error for %s coin %s: %v", fundingAsset.Symbol, dexCoin, err)
+		return msgjson.NewError(msgjson.FundingError, fmt.Sprintf("failed to verify coin %v", dexCoin))
```

Comment:
I believe the `checkZeroConfs` calls in `handleLimit` and `handleMarket` will need to be moved into the TryFunc given to the `latencyQ` waiter's `TryFunc`.  Otherwise it'll reject txsplit orders pretty reliably, esp. with realistic network conditions.

Reply:
OMG, misread, sorry!